src/components/NavigationButton.tsx

```diff
@@ -1,24 +1,56 @@
 import { Button } from '@mui/material';
 import useViewStore from '../stores/view';
+import { View } from '../utils/types';
+
+const defaultProps = {
+  backLabel: 'Back',
+  nextLabel: 'Next',
+  className: '',
+};
 
 function NavigationButton({
-  label,
-  viewToNavigateTo,
+  backView,
+  nextView,
+  backLabel,
+  nextLabel,
+  className,
 }: {
-  label: string;
-  viewToNavigateTo: string;
+  backView: View | undefined;
+  nextView: View | undefined;
+  backLabel?: string;
+  nextLabel?: string;
+  className?: string;
```

Comment:
```suggestion
  tailwindClassName?: string;
```
or something else that makes it very clear this is about styling